src/App.jsx

```diff
@@ -20,6 +20,10 @@ function App() {
             path="/articles/:article_id/comments"
             element={<SingleArticle />}
           />
+          <Route
```

Comment:
You have this twice it seems, also it may not need to be on an another route, you could just render the comments component on the single article page think about how things like youtube would have the main content then the comments section underneath